2020-10-08 12:33:46,729 INFO 55260 [egg-logrotator] agent logger reload: got log-reload message
2020-10-08 22:51:47,198 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 574984ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-08 23:02:05,008 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 577802ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-08 23:12:00,422 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575405ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-08 23:22:15,794 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575350ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-08 23:32:10,750 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 574946ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-08 23:42:26,136 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575376ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-08 23:52:22,108 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575962ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 00:02:37,449 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575323ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 00:12:32,804 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575346ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 00:22:47,714 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 574897ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 00:32:43,116 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575389ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 00:42:59,505 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 576379ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 00:52:54,842 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575326ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 01:02:51,002 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 576157ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 01:13:06,619 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575600ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 01:23:02,027 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575397ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 01:33:17,331 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575287ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 01:43:12,570 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575231ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 01:53:27,732 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575150ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 02:03:25,989 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 578250ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 02:13:41,362 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575366ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 02:23:36,820 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575447ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 02:33:52,094 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575254ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 02:43:46,953 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 574846ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 02:54:02,274 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575306ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 03:03:58,397 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 576111ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 03:14:13,614 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575198ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 03:24:08,831 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575211ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 03:34:24,258 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575404ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 03:44:22,770 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 578496ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 03:54:38,062 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575274ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 04:04:33,414 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575343ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 04:14:48,947 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575520ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 04:24:44,338 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575376ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 04:35:00,078 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575728ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 04:44:55,312 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575221ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 04:55:13,705 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 578379ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 05:05:08,909 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575192ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 05:15:24,743 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575819ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 05:25:19,056 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 574307ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 05:35:16,003 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 576939ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 05:45:31,793 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575770ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 05:55:26,846 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575047ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 06:05:42,100 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575239ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 06:15:37,590 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575485ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 06:25:53,040 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575437ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 06:35:48,553 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575503ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 06:46:03,872 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575312ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 06:55:59,385 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575503ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 07:06:14,694 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575294ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 07:16:10,107 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575399ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 07:26:25,519 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575394ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 07:36:20,867 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575334ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 07:46:36,838 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575956ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 07:56:32,080 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575232ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 08:06:47,396 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575303ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 08:16:42,687 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575278ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 08:26:58,130 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575428ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 08:36:53,516 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 575374ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 08:46:50,001 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 576478ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 09:54:59,407 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 4049395ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 09:58:03,405 ERROR 55260 nodejs.ClusterClientNoResponseError: client no response in 163989ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/lxd/Desktop/mine/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 55260
hostname: localhost

2020-10-09 10:12:04,700 WARN 658 Sequelize Error: read ECONNRESET, sleep 1 seconds to retry...
2020-10-09 10:12:03,751 INFO 658 [egg:logger] init all loggers with options: {"dir":"/Users/lxd/Desktop/mine/egg-example/logs/egg-demo","encoding":"utf8","env":"local","level":"INFO","consoleLevel":"INFO","disableConsoleAfterReady":false,"outputJSON":false,"buffer":true,"appLogName":"egg-demo-web.log","coreLogName":"egg-web.log","agentLogName":"egg-agent.log","errorLogName":"common-error.log","coreLogger":{"consoleLevel":"WARN"},"allowDebugAtProd":false,"type":"agent"}
2020-10-09 10:12:04,079 INFO 658 [egg:core] dump config after load, 4ms
2020-10-09 10:12:04,119 INFO 658 [egg-watcher] Start watching: ["/Users/lxd/Desktop/mine/egg-example/app","/Users/lxd/Desktop/mine/egg-example/config","/Users/lxd/Desktop/mine/egg-example/mocks","/Users/lxd/Desktop/mine/egg-example/mocks_proxy","/Users/lxd/Desktop/mine/egg-example/app.js"]
2020-10-09 10:12:04,119 INFO 658 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/app"
2020-10-09 10:12:04,120 INFO 658 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/config"
2020-10-09 10:12:04,120 INFO 658 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/mocks"
2020-10-09 10:12:04,120 INFO 658 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/mocks_proxy"
2020-10-09 10:12:04,120 INFO 658 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/app.js"
2020-10-09 10:12:04,121 INFO 658 [egg-watcher:agent] watcher start success
2020-10-09 10:12:05,706 WARN 658 Sequelize Error: connect ECONNREFUSED 127.0.0.1:3306, sleep 1 seconds to retry...
2020-10-09 10:12:06,712 WARN 658 Sequelize Error: connect ECONNREFUSED 127.0.0.1:3306, sleep 1 seconds to retry...
2020-10-09 10:12:07,714 ERROR 658 nodejs.SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:3306
    at ConnectionManager.connect (/Users/lxd/Desktop/mine/egg-example/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:116:17)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
name: "SequelizeConnectionRefusedError"
parent: {"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":3306,"fatal":true}
original: {"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":3306,"fatal":true}
pid: 658
hostname: lxddeMacBook-Pro.local

2020-10-09 10:12:07,720 INFO 658 [egg:core] dump config after ready, 5ms
2020-10-09 10:13:20,747 WARN 765 Sequelize Error: connect ECONNREFUSED 127.0.0.1:3306, sleep 1 seconds to retry...
2020-10-09 10:13:20,558 INFO 765 [egg:logger] init all loggers with options: {"dir":"/Users/lxd/Desktop/mine/egg-example/logs/egg-demo","encoding":"utf8","env":"local","level":"INFO","consoleLevel":"INFO","disableConsoleAfterReady":false,"outputJSON":false,"buffer":true,"appLogName":"egg-demo-web.log","coreLogName":"egg-web.log","agentLogName":"egg-agent.log","errorLogName":"common-error.log","coreLogger":{"consoleLevel":"WARN"},"allowDebugAtProd":false,"type":"agent"}
2020-10-09 10:13:20,719 INFO 765 [egg:core] dump config after load, 4ms
2020-10-09 10:13:20,743 INFO 765 [egg-watcher] Start watching: ["/Users/lxd/Desktop/mine/egg-example/app","/Users/lxd/Desktop/mine/egg-example/config","/Users/lxd/Desktop/mine/egg-example/mocks","/Users/lxd/Desktop/mine/egg-example/mocks_proxy","/Users/lxd/Desktop/mine/egg-example/app.js"]
2020-10-09 10:13:20,743 INFO 765 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/app"
2020-10-09 10:13:20,744 INFO 765 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/config"
2020-10-09 10:13:20,744 INFO 765 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/mocks"
2020-10-09 10:13:20,744 INFO 765 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/mocks_proxy"
2020-10-09 10:13:20,744 INFO 765 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/app.js"
2020-10-09 10:13:20,744 INFO 765 [egg-watcher:agent] watcher start success
2020-10-09 10:13:21,750 WARN 765 Sequelize Error: connect ECONNREFUSED 127.0.0.1:3306, sleep 1 seconds to retry...
2020-10-09 10:13:22,752 WARN 765 Sequelize Error: connect ECONNREFUSED 127.0.0.1:3306, sleep 1 seconds to retry...
2020-10-09 10:13:23,758 ERROR 765 nodejs.SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:3306
    at ConnectionManager.connect (/Users/lxd/Desktop/mine/egg-example/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:116:17)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
name: "SequelizeConnectionRefusedError"
parent: {"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":3306,"fatal":true}
original: {"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":3306,"fatal":true}
pid: 765
hostname: lxddeMacBook-Pro.local

2020-10-09 10:13:23,765 INFO 765 [egg:core] dump config after ready, 6ms
2020-10-09 10:17:00,340 WARN 1088 Sequelize Error: connect ECONNREFUSED 127.0.0.1:3306, sleep 1 seconds to retry...
2020-10-09 10:17:00,151 INFO 1088 [egg:logger] init all loggers with options: {"dir":"/Users/lxd/Desktop/mine/egg-example/logs/egg-demo","encoding":"utf8","env":"local","level":"INFO","consoleLevel":"INFO","disableConsoleAfterReady":false,"outputJSON":false,"buffer":true,"appLogName":"egg-demo-web.log","coreLogName":"egg-web.log","agentLogName":"egg-agent.log","errorLogName":"common-error.log","coreLogger":{"consoleLevel":"WARN"},"allowDebugAtProd":false,"type":"agent"}
2020-10-09 10:17:00,312 INFO 1088 [egg:core] dump config after load, 4ms
2020-10-09 10:17:00,335 INFO 1088 [egg-watcher] Start watching: ["/Users/lxd/Desktop/mine/egg-example/app","/Users/lxd/Desktop/mine/egg-example/config","/Users/lxd/Desktop/mine/egg-example/mocks","/Users/lxd/Desktop/mine/egg-example/mocks_proxy","/Users/lxd/Desktop/mine/egg-example/app.js"]
2020-10-09 10:17:00,335 INFO 1088 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/app"
2020-10-09 10:17:00,336 INFO 1088 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/config"
2020-10-09 10:17:00,336 INFO 1088 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/mocks"
2020-10-09 10:17:00,336 INFO 1088 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/mocks_proxy"
2020-10-09 10:17:00,336 INFO 1088 [egg-watcher] Start watching: "/Users/lxd/Desktop/mine/egg-example/app.js"
2020-10-09 10:17:00,336 INFO 1088 [egg-watcher:agent] watcher start success
2020-10-09 10:17:01,869 WARN 1088 Sequelize Error: read ECONNRESET, sleep 1 seconds to retry...
2020-10-09 10:17:02,875 WARN 1088 Sequelize Error: connect ECONNREFUSED 127.0.0.1:3306, sleep 1 seconds to retry...
2020-10-09 10:17:03,879 ERROR 1088 nodejs.SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:3306
    at ConnectionManager.connect (/Users/lxd/Desktop/mine/egg-example/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:116:17)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
name: "SequelizeConnectionRefusedError"
parent: {"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":3306,"fatal":true}
original: {"errno":"ECONNREFUSED","code":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":3306,"fatal":true}
pid: 1088
hostname: lxddeMacBook-Pro.local

2020-10-09 10:17:03,887 INFO 1088 [egg:core] dump config after ready, 7ms
